paint/preprocessing/focal_spot_extractor.py

```diff
@@ -0,0 +1,291 @@
+import json
+from pathlib import Path
+from typing import Union
+
+import cv2
+import torch
+
+import paint.util.paint_mappings as mappings
+
+
+class FocalSpot:
+    """
+    Data class for storing focal spot information.
+
+    Attributes
+    ----------
+    flux : torch.Tensor
+        Intensity image of the detected focal spot.
+    aim_point_image : tuple[float, float]
+        Detected center of intensity in image coordinates (height, width).
```

Comment:
Just asking out of curiosity: Why is this a tuple of floats and not also a tensor?